templates/graphql-cloudflare-analytics-sdl/grafbase/.env

```diff
@@ -0,0 +1 @@
+API_TOKEN=
```

Comment:
Newline, do we not want a default value here? Not sure how it'd behave with this